README.md

```diff
@@ -155,9 +123,8 @@ the following structure:
             \-- k8s-objects
 ```
 
-`k8s-objects-generator` will invoke both the `go` and the `easyjson` binaries
-using `~/k8s-data-types` as exclusive `GOPATH`. That means that, by the end of
-the process, the output directory will feature also `pkg` and other `src`
+`k8s-objects-generator` will invoke the `go` binary using `~/k8s-data-types` as exclusive `GOPATH`.
```

Comment:
Is this structure still relevant?

Reply:
I think we still need to do that in order to generate the `go.mod` and `go.sum` files.